sld407-matter-ota/index.md

```diff
@@ -4,5 +4,5 @@ The Over The Air (OTA) Software Update functionality is enabled by default for a
 
 - [**Matter OTA Bootloader**](./01-ota-bootloader.md)
 - [**Matter OTA Software Update**](./02-ota-software-update.md)
-- [**Matter 917 SOC OTA Software Update**](./04-ota-software-update-soc.md)
-- [**Matter OTA WiFi Project**](./05-build-ota-application-using-studio.md)
+- [**Matter 917 SOC OTA Software Update**](./03-ota-software-update-soc.md)
+- [**Wi-Fi-Based Matter OTA Project in Simplicity Studio**](./04-build-ota-application-using-studio.md)
```

Comment:
Looks like there is nothing WiFi-specific in the  04-build-ota-application-using-studio.md document and even the title does not mention WiFi. If so, perhaps we should remove "Wi-Fi-Based" from the link text?